Backend/ItemService/src/item/item.manager.ts

```
@@ -0,0 +1,9 @@
+import { Item } from "@Libs/types/DB/item.type";
+import { ItemRepository } from "./item.repository";
+
+export const ItemManager = {
+  getAll: async (): Promise<Item[]> => await ItemRepository.getAll(),
+
+  getItemById: async (id: string): Promise<Item | null> =>
+    await ItemRepository.getItemById(id),
```

wouldn't you want to throw a 404 error if a user is not found?